core/pom.xml

```diff
@@ -101,6 +101,11 @@ THE SOFTWARE.
       </exclusions>
     </dependency>
 
+    <dependency>
```

Comment:
There is a dependency on `jbcrypt` in Jenkins Core so we have to add the dependency

```
/home/jenkins/workspace/Core_jenkins_PR-4085/core/src/main/java/hudson/security/HudsonPrivateSecurityRealm.java:[68,26] error: package org.mindrot.jbcrypt does not exist
```

Reply:
FTR, from current `master`:

```
$ mvn -f core dependency:tree
…
[INFO] +- org.jenkins-ci:trilead-ssh2:jar:build-217-jenkins-14:compile
[INFO] |  \- org.connectbot.jbcrypt:jbcrypt:jar:1.0.0:compile
…
```